ui/src/timeMachine/actions/queries.ts

```diff
@@ -148,10 +155,14 @@ export const executeQueries = () => async (dispatch, getState: GetState) => {
 
       const extern = buildVarsOption(variableAssignments)
 
-      return runQuery(orgID, text, extern)
+      return runQuery(orgID, text, extern, abortController)
     })
+    const results = await Promise.all(pendingResults.map(r => r.promise)).catch(
+      error => {
```

Comment:
I _think_ if we are catching and then throwing error, it's the same as not catching at all?

Reply:
That's a great question, you could be right. Let me go ahead and give that another shot. I think I originally added it to help the error bubble out of the promise, but it could be that I had an issue in the catch at the end of all this

Reply:
Yup, you're totally right. I will ✂️  it out